courses/free-courses-hi.md

```diff
@@ -196,6 +197,12 @@
 * [Machine Learning Tutorials For Beginners Using Python in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ai6fAMHp-acBmJONT7Y4BSG) - CodeWithHarry (YouTube)
 
 
+### Operating Systems
+
+* [Operating System(Complete Playlist)](https://www.youtube.com/playlist?list=PLxCzCOWd7aiGz9donHRrE9I3Mwn6XdP8p) - Gate Smashers (YouTube)
+* [Operating Systems for Placements](https://www.youtube.com/playlist?list=PLDzeHZWIZsTr3nwuTegHLa2qlI81QweYG) - CodeHelp-byBabbar (YouTube)
```

Comment:
title is 'Operating Systems Placements Course 2022'

Reply:
thanks sir, but the given title is 'Operating Systems Placements Course 2022' so i updated old one with it.